frontend/src/pages/modelServing/screens/global/ModelServingGlobal.stories.tsx

```diff
@@ -143,3 +143,58 @@ DeleteModel.play = async ({ canvasElement }) => {
   await userEvent.type(retypeNameInput, 'Test Inference Service', { delay: 50 });
   expect(deleteButton).not.toBeDisabled();
 };
+
+export const DeployModel = Template.bind({});
+DeployModel.parameters = {
+  a11y: {
+    // need to select modal as root
+    element: '.pf-c-backdrop',
+  },
+};
+DeployModel.play = async ({ canvasElement }) => {
+  // load page and wait until settled
+  const canvas = within(canvasElement);
+  await canvas.findByText('Test Inference Service', undefined, { timeout: 5000 });
+
+  // user flow for editing a project
+  await userEvent.click(canvas.getByText('Deploy model', { selector: 'button' }));
+
+  // get modal
+  const body = within(canvasElement.ownerDocument.body);
+  const nameInput = body.getByRole('textbox', { name: 'Model Name' });
+  const updateButton = body.getByText('Deploy', { selector: 'button' });
+
+  // test that you can not submit on empty
+  await userEvent.clear(nameInput);
```

Comment:
Why do you clear it? It's already empty -- this could conceal a bug about data being left over from a previous iteration (which is a bug we have had). It probably doesn't matter in this iteration of the test suite, but as we move forward, we should avoid doing "things we know are already done" to help with catching gaps.